test/graphql/types/Organization/creator.test.ts

```
@@ -0,0 +1,459 @@
+import type { FastifyInstance, FastifyReply } from "fastify";
+import type { MercuriusContext } from "mercurius";
+import {
+  beforeEach,
+  describe,
+  expect,
+  it,
+  vi,
+  MockedFunction as viMockedFunction,
+} from "vitest";
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)